pkg/addon/helper.go

```diff
@@ -127,15 +128,59 @@ func EnableAddonByLocalDir(ctx context.Context, name string, dir string, cli cli
 	return nil
 }
 
-// GetAddonStatus is genrall func for cli and apiServer get addon status
+// GetAddonStatus is a wrapper for GetAddonStatusDetailed, to provide backwards compatibility.
+// It is faster than GetAddonStatusDetailed, and should be used in most cases.
```

Comment:
Why is it faster?

Reply:
Because it is equivalent to `GetAddonStatusDetailed(ctx, cli, name, false)`, notice the last argument.

Setting the last argument `detailed` to `true` will add `AddonPackage` (the addon install package, which includes parameter descriptions, default vaues and other information) to the return value. These information will be be fetched from the registry, which will take some time. 

So, if `AddonPackage` is not needed by the caller, setting the last argument `detailed` to `false` will skip fetching addon install packages, to speed things up

Because this function is called many times when listing addons `addon list` (we need to check each addon is enabled), it makes sense to make this faster.

Reply:
Of course, it also makes sense to take the logic to get the addon install package outside of `func GetAddonStatus`. Then we don't really need these *faster* equivalents. Just use a separate function when needed.

It is probably a better way? I will refactor the code later.